src/main/java/htsjdk/variant/vcf/VCFAltHeaderLine.java

```diff
@@ -0,0 +1,75 @@
+package htsjdk.variant.vcf;
+
+import htsjdk.samtools.util.Log;
+import htsjdk.tribble.TribbleException;
+
+import java.util.*;
+
+//TODO: Should we validate these alt allele types ?
```

Comment:
I agree that sounds right, but it could break backward compatibility. So perhaps we should only do it starting with v4.3.